WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewController.swift

```diff
@@ -363,7 +367,8 @@ private extension ReviewsViewController {
     ///
     @objc func defaultSiteWasUpdated() {
         reloadResultsController()
-        syncingCoordinator.resetInternalState()
+        syncingCoordinator.resynchronize()
+        tableView.reloadData()
```

Comment:
nit: is this call necessary? I commented this line and it seems to work still

Reply:
I confirm that it works properly also without this `reloadData()`. Removed here 3ddb199db0e0e2eaf0204e6729e710c3d1159409 🙇